docs/source/usage/plugins/countPerSupercell.rst

```diff
@@ -3,7 +3,7 @@
 Count per Supercell
 -------------------
 
-This plugin counts the total number of *macro particles of a species* for each super cell and sstores the result in an hdf5 file. 
+This plugin counts the total number of *macro particles of a species* for each super cell and stores the result in an hdf5 file. 
 Only in case of constant particle density, where each macro particle describes the same number of real particles (weighting), conclusions on the plasma density can be drawn.
```

Comment:
Isn't the important thing only constant weight, and plasma density is unrelated? Or `constant particle density` is supposed to mean the constant weight?

Reply:
Let's write: `Only in case of constant particle weighting, where each macro particle describes the same number of real particles, conclusions on the plasma density can be drawn.`

The doc implicitly assumes that we distribute our particles equally, which we currently always do. So both is correct, but we can make it more explicit.

Reply:
Done, #2708  . I guess the same applies to my earlier comment in this PR about hyphenation of floating-point (currently written as separate words), or you disagree with that?

Reply:
Fixed as @ax3l suggested.